server/src/test/java/io/javaclasses/fileHub/web/routes/GetFolderContentRouteTest.java

```diff
@@ -0,0 +1,186 @@
+package io.javaclasses.fileHub.web.routes;
+
+import com.google.common.testing.NullPointerTester;
+import io.javaclasses.filehub.api.getFolderContentView.FileMimeType;
+import io.javaclasses.filehub.api.getFolderContentView.FileSize;
+import io.javaclasses.filehub.api.getFolderContentView.FileType;
+import io.javaclasses.filehub.storage.fileSystemItemsStorage.*;
+import io.javaclasses.filehub.storage.loggedInUsersStorage.LoggedInUserRecord;
+import io.javaclasses.filehub.storage.loggedInUsersStorage.Token;
+import io.javaclasses.filehub.storage.userStorage.UserId;
+import io.javaclasses.filehub.web.CurrentUser;
+import io.javaclasses.filehub.web.ServerTimeZone;
+import io.javaclasses.filehub.web.routes.GetFolderContentRoute;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import spark.Request;
+import spark.Response;
+
+import java.time.LocalDate;
+
+import static javax.servlet.http.HttpServletResponse.SC_NOT_FOUND;
+import static javax.servlet.http.HttpServletResponse.SC_OK;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.fail;
+
+@DisplayName("GetFolderContentRoute should: ")
+public class GetFolderContentRouteTest {
+
+    @DisplayName("process valid requests.")
+    @Test
+    public void validRequestsTest() {
+        LoggedInUserRecord userRecord = createLoggedInUserRecord();
+        CurrentUser.set(userRecord);
+
+        FolderStorage folderStorage = createFolderStorage();
+        FileStorage fileStorage = createFileStorage();
+        int numberOfElements = 3;
+        FileSystemItemId parentId = createParentId();
+        UserId ownerId = createOwnerId();
+        prepareStorage(fileStorage, folderStorage, numberOfElements, parentId, ownerId);
+
+        GetFolderContentRoute route = new GetFolderContentRoute(folderStorage, fileStorage);
+
+        Request request = createMockRequest();
+        Response response = createMockResponse();
+
+        try {
+
+            Object handle = route.handle(request, response);
+
+            assertNotNull(handle, "Handle method should return not null value.");
+            assertEquals(SC_OK, response.status(),
+                    "Handle method does not throw exception when request is valid.");
+
+        } catch (Exception exception) {
+
+            fail("GetFolderContentRoute should not throw any exceptions. Exception: " + exception.getClass());
+        }
+    }
+
+    @DisplayName("return error responses when requests are invalid.")
```

Comment:
Request can not be invalid, please consider:
```suggestion
    @DisplayName("return error responses when requested folder is not found.")
```
or smth like that.